app.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const getTopics = require('./controllers/topics-controller')
+
+const app = express()
+
+app.get('/api/topics', getTopics)
+
+app.use((err, req, res, next) => {
+    if (err.status && err.msg) {
+      res.status(err.status).send({ msg: err.msg });
+    } else if (err.code) {
+      res.status(400).send({ msg: "Bad Request" });
+    }
```

Comment:
Please make sure that you are testing for specific sql error codes e.g. 22P02 so that you can send back a bespoke error message :)